primap2/_downscale.py

```diff
@@ -65,9 +68,12 @@ def downscale_timeseries(
         basket_contents_da = da_sel.loc[{dim: basket_contents}]
         basket_da = da_sel.loc[{dim: basket}]
 
-        basket_sum = basket_contents_da.pr.sum(
-            dim=dim, skipna_evaluation_dims=skipna_evaluation_dims
-        )
+        if (skipna_evaluation_dims is not None) or (skipna is None) or not skipna:
+            basket_sum = basket_contents_da.pr.sum(
+                dim=dim, skipna_evaluation_dims=skipna_evaluation_dims
+            )
+        else:
+            basket_sum = basket_contents_da.pr.sum(dim=dim, skipna=True, min_count=1)
```

Comment:
yes it should. I just have to make sure that the parameters are set correctly such that sum does the right thing. But better than duplicating code

Reply:
sum currently has a better treatment of the different nan-treatment parameters as it throws an error if both are set. This doesn't work well with the default values in the downscaling functions as it means that you have to set skipna explicitly to None if you want to use skipna_evaluation_dims.

Reply:
hmm, a possible solution would be to actually change sum to silently discard `skipna` if `skipna_evaluation_dims` is given. I think the intent of the user is pretty clear. If I say "skip NA values, but only check the time dimension" (which is what I would interpret `skipna=True` and `skipna_evaluation_dims=["time"]` as), then just disregarding `skipna` is correct. What do you think?

Reply:
Maybe we need to talk again about usual aggregation. Is `skipna_evaluation_dims` even used`? I implemented it because I thought that primap1 actually behaved as if `skipna_evaluation_dims=["time"]`. But as I understand now, primap1 behaves as if `skipna=True` and `min_count=1`, right?

`skipna_evaluation_dims` is not the easiest thing to use or maintain. If it is not actually needed for a composite source generator in primap2, I'm also more than happy to drop it again.

Reply:
I think I do use it when downscaling data. Will double check tomorrow.

Reply:
So, a bit later than tomorrow, but I checked. I currently don't use skipna_evaluation_dims

Reply:
I've implemented a solution that has only one call to 'ds.pr.sum()' for all cases of skipna and skipna_evaluation_dims. The code is not shorter though as I added code to raise an error if both are set and to set 'skipna' to 'None' if 'skipna_evaluation_dims' is not 'None' and 'skipna=False'. 'min_count' is always passed to 'ds.pr.sum' as '1', but unused if 'skipna_evaluation_dims' is used.



      if skipna_evaluation_dims is not None:
            if skipna:
                raise ValueError(
                    "Only one of 'skipna' and 'skipna_evaluation_dims' may be supplied, not"
                    " both."
                )
            else:
                skipna = None

        basket_sum = basket_contents_converted.pr.sum(
                dim="entity", skipna=skipna, min_count=1,
            skipna_evaluation_dims=skipna_evaluation_dims)

If we decide that we have no use for 'skipna_evaluation_dims' we can still remove it. I would keep it for now until we have more uses cases of PRIMAP2